tensorboard/backend/event_processing/db_import_multiplexer_test.py

```diff
@@ -19,49 +19,33 @@
 
 import os
 import os.path
+import sqlite3
 
-from tensorboard.backend import application
 from tensorboard.backend.event_processing import db_import_multiplexer
-from tensorboard.compat.proto import event_pb2
-from tensorboard.compat.proto import summary_pb2
-from tensorboard.compat.proto import tensor_pb2
-from tensorboard.compat.proto import types_pb2
 import tensorflow as tf
 
 
-def _AddEvents(path):
+def add_event(path):
   with tf.summary.FileWriter(path) as writer:
```

Comment:
FWIW, didn't notice this before, but this will actually create a new FileWriter (and corresponding new event file) each time you call add_event() which is a bit unnecessary.

Instead, you could either create the writer once per test case in setUp(), or you could use FileWriterCache.